modules/workflow-service-api/src/main/java/org/opencastproject/workflow/api/WorkflowInstanceImpl.java

```diff
@@ -88,6 +93,9 @@
   @XmlElement(name = "organization", namespace = "http://org.opencastproject.security")
   private JaxbOrganization organization;
 
+  @XmlElement(name = "organizationid", namespace = "http://org.opencastproject.security")
```

Comment:
Maybe organization_id or organizationId for sake of readability?

Reply:
I changed it to `organization-id` to be consistent with other XML tags we use like for example `time-in-queue`